_maps/RandomRuins/SpaceRuins/Immortus.dmm

```diff
@@ -0,0 +1,3406 @@
+/turf/closed/indestructible/vault,
+/area/ruin/powered/Immortus)
+"cY" = (
+/obj/machinery/light/sequence{
+	dir = 1;
+	group = 4
+	},
+/turf/open/floor/plasteel/dark,
+/area/ruin/powered/Immortus)
+"cZ" = (
+/obj/machinery/light/sequence{
+	dir = 1;
+	group = 3
+	},
+/turf/open/floor/plasteel/dark,
+/area/ruin/powered/Immortus)
+"da" = (
+/obj/machinery/light/sequence{
+	dir = 1;
+	group = 2
+	},
+/turf/open/floor/plasteel/dark,
+/area/ruin/powered/Immortus)
+"db" = (
+/obj/machinery/light/sequence{
+	dir = 1
+	},
+/turf/open/floor/plasteel/dark,
+/area/ruin/powered/Immortus)
+"dc" = (
+/obj/machinery/light/sequence{
+	dir = 1;
+	group = 8
+	},
+/turf/open/floor/plasteel/dark,
+/area/ruin/powered/Immortus)
+"dd" = (
+/obj/structure/statue/haunted,
+/turf/open/floor/plasteel/dark,
+/area/ruin/powered/Immortus)
+"de" = (
+/obj/machinery/light/sequence{
+	dir = 8;
+	group = 9
+	},
+/turf/open/floor/plasteel/dark,
+/area/ruin/powered/Immortus)
+"df" = (
+/obj/machinery/light/sequence{
+	dir = 1;
+	group = 6
+	},
+/turf/open/floor/plasteel/dark,
+/area/ruin/powered/Immortus)
+"dg" = (
+/obj/structure/statue/haunted/corgi,
+/turf/open/floor/plasteel/dark,
+/area/ruin/powered/Immortus)
+"dh" = (
+/obj/machinery/door/airlock/public/glass,
+/turf/open/floor/plasteel/dark,
+/area/ruin/powered/Immortus)
+"di" = (
+/obj/structure/barricade/wooden{
+	max_integrity = 40
+	},
+/turf/open/floor/plasteel/dark,
+/area/ruin/powered/Immortus)
+"dj" = (
+/obj/structure/statue/haunted/female,
+/turf/open/floor/plasteel/dark,
+/area/ruin/powered/Immortus)
+"dk" = (
+/obj/machinery/door/poddoor/preopen,
+/obj/machinery/door/airlock/vault,
+/turf/open/floor/plasteel/dark,
+/area/ruin/powered/Immortus)
+"dl" = (
+/obj/effect/spawner/structure/window/shuttle,
+/turf/open/floor/plasteel/dark,
+/area/ruin/powered/Immortus)
+"dm" = (
+/obj/structure/table/wood/bar,
+/obj/item/paper/contract/immortus{
+	pixel_x = 6;
+	pixel_y = 7
+	},
+/obj/item/device/camera/spooky,
+/turf/open/floor/plasteel/dark,
+/area/ruin/powered/Immortus)
+"dn" = (
+/obj/machinery/light/sequence{
+	group = 4
+	},
+/turf/open/floor/plasteel/dark,
+/area/ruin/powered/Immortus)
+"do" = (
+/obj/structure/table/wood/bar,
+/obj/item/spellbook/oneuse/weakpetrify,
+/obj/machinery/light/small{
+	dir = 8
+	},
+/turf/open/floor/plasteel/dark,
+/area/ruin/powered/Immortus)
+"dp" = (
+/obj/machinery/light/sequence{
+	dir = 2;
+	group = 7
+	},
+/turf/open/floor/plasteel/dark,
+/area/ruin/powered/Immortus)
+"dq" = (
+/obj/machinery/light/sequence{
+	dir = 8;
+	group = 3
+	},
+/turf/open/floor/plasteel/dark,
+/area/ruin/powered/Immortus)
+"dr" = (
+/obj/machinery/light/sequence{
+	dir = 4
+	},
+/turf/open/floor/plasteel/dark,
+/area/ruin/powered/Immortus)
+"ds" = (
+/obj/machinery/light/sequence{
+	dir = 2;
+	group = 5
+	},
+/turf/open/floor/plasteel/dark,
+/area/ruin/powered/Immortus)
+"dt" = (
+/obj/machinery/light/sequence{
+	group = 2
+	},
+/obj/structure/table/reinforced,
+/obj/item/spellbook{
```

Comment:
add persistence_replacement = null, otherwise SSpersistence will replace it with a random real spellbook.